### ISSUE TITLE
1.19 version and prerelease version both give incorrect seed in Realm in 1.19.2

### ISSUE THREAD

**Issue by DimensionalDrift1** [19MisterX98/SeedcrackerX#127]
I was trying to crack the seed for a vanilla 1.19.2 realm that I'm a part of. Each time it's "found the world seed", it shows the same, albeit incorrect, seed.  Is there anything I'm doing wrong?

Things I've tried:
-Using 1.19 version, and prerelease version
-Disabling everything except shipwrecks, and only using those
-Going to different areas of the world and trying there
-Restarting client multiple times.

Each time, I've gotten the exact same seed, and checking both on chunkbase and in singleplayer, it's not correct. I would like to reiterate that this realm is vanilla.

**Comment by 19MisterX98** [comment]
Maybe the realm uses the large biome setting

**Comment by DimensionalDrift1** [comment]
Just checked chunkbase for large biomes, and also created a large biome world, amplified too, and none of them are correct. Is it possible that realms uses a different world gen?

**Comment by DimensionalDrift1** [comment]
As an update; I'm even more confused now.

I managed to get the seed from the guy who owns the realm. It's the ***same*** seed as the one the seed cracker gave me. This felt very weird, so I checked chunkbase for the same seed in different versions, and found that the 1.17 had similar biomes to the realm. As one would expect, I loaded up 1.17 and generated a world with the same seed, and this is where I get confused.

I went to the coordinates of my base, lo and behold, there's the *exact* same mountain that I have my base on. All of the surrounding terrain is somewhat different; some of the biomes are wrong, there's a village missing as well.

I was wondering if you have an explanation for this. Clearly it's not an issue with the mod, so I'll be closing the issue, but I'm still sorely confused, and I hope that chunkbase will still help me with slime chunks.
